graylog2-web-interface/src/components/outputs/OutputList.jsx

```diff
@@ -33,6 +33,7 @@ class OutputList extends React.Component {
     types: PropTypes.object.isRequired,
   };
 
+  // eslint-disable-next-line class-methods-use-this
   _sortByTitle = (output1, output2) => {
     return naturalSort(output1.title.toLowerCase(), output2.title.toLowerCase());
   };
```

Comment:
We can just move `sortByTitle` outside of the component and avoid using `eslint-disable`.